tests/system.py

```diff
@@ -736,6 +736,65 @@ def test_load_table_from_dataframe_w_automatic_schema(self):
         )
         self.assertEqual(table.num_rows, 3)
 
+    @unittest.skipIf(
+        pandas is None or pandas.__version__ < "1.0.1",
```

Comment:
Thought - I _think_ comparing plain version strings will actually work in this particular case, but cannot be generalized (e.g. somebody copies this pattern over to another test, or if we have to change the version for some reason).

Let's instead add a library like [packaging](https://pypi.org/project/packaging/) as a test dependency, and use that for parsing and semantically comparing versions?

Reply:
BTW, why pandas version `1.0.1+`, as the code sample from the issue description also seems to works in pandas 1.0.0?

Reply:
Agreed, but need suggestion on version comparison like should i use `parse(pandas.__version__) < parse("1.0.0")` or should i change the pandas required minimum version in [setup.py](https://github.com/googleapis/python-bigquery/blob/master/setup.py#L47) file  and skip that if PY2 or something else?

Reply:
Since the issue was due to a bug in Pandas, it's probably best to directly compare the versions of Pandas rather than a proxy value such as the Python version.

Re: bumping the version - if the new Pandas version is still compatible with Python 2.7, we can bump it, otherwise we will have to wait until April or so when we officially drop Python 2 support (IIRC).

Reply:
New version of pandas isn't compatible with Python 2.7, it Requires: Python >=3.6.1.

Reply:
I see. We could still conditionally bump the pandas version for Python 3.5+ in `setup.py`, though, to at least guarantee correct behavior in Python 3, even though we cannot fix the bug in Python 2.7 due to pandas incompatibility.

@tswast What do you think?

P.S.: If there is an efficient way of detecting whether a column contains a `NaN` value, we could also emit a user warning if too old a pandas version is installed... just a thought.

Reply:
@tswast bump

Reply:
We have setuptools as a required dependency, so why aren't you using `pkg_resources`?

https://github.com/pydata/pandas-gbq/blob/97e9a9ee3a8f65ede29f25954b1af3f3bc606f58/pandas_gbq/gbq.py#L46

Reply:
A lot of places (including Google) are pretty slow to update pandas, so I'd hesitate to require 1.0+ just yet.